lib/test_result.dart

```diff
@@ -75,6 +76,88 @@ class TestResult {
     return TestAttachment.fromJson(response);
   }
 
+  static Future<TestResults> getTestRunSingleTestResults({
+    required int testId,
```

Comment:
I don't quite understand the `TestRun` qualifier here
```suggestion
  static Future<TestResults> getTestResults(
    int testId, {
```

Reply:
It's because, in TestRun, you can get specific Test results which are "deep copy" from TestCase. In fact, that's quite confusing. 
(API reference #getresults)[https://www.gurock.com/testrail/docs/api/reference/results/#getresults]